### DISCUSSION TITLE
Colors

### DISCUSSION THREAD

**Discussion by hugoocoto** [iamcco/markdown-preview.nvim#698] [Ideas]
It would be great if I can customize colors, I use it for studying and I prefer colorized text instead of pure B&W. I mean, I have my nvim configured to show different colors in headers, bold, italic... And I want to make web preview look close to it. Thanks

**Comment by aspauldingcode** [comment]
Huh... That's not how markdown works... 

But fascinating idea. I wonder if css could be used somewhere. 
Maybe you could try scripting up a Tampermonkey script which injects the preview locally and replaces headers colors, text body colors, etc.
